docs/hosting/hetzner.md

```
@@ -0,0 +1,434 @@
+```
+
+```bash
+# Make backup script executable
+chmod +x /opt/sure/backup.sh
+
+# Add to crontab for daily backups at 2 AM
+crontab -e
+```
+
+Add this line to crontab:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)